admin_manual/configuration_server/performance_tuning.rst

```diff
@@ -34,11 +54,32 @@ performance. To do so, add the following to ``/etc/fstab``::
 
 	none /tmp tmpfs,size=6g defaults
 
-Make sure the APC byte code cache is installed:
+Make sure the APC or Opcache bytecode cache is installed. This example is for 
+CentOS/Red Hat/Fedora running PHP 5.4:
+
+.. code-block:: console
+
+	$ sudo yum install php-pecl-apc
+	
+On Ubuntu systems running PHP 5.4 this command installs APC:
+
+.. code-block:: console
+
+        $ sudo apt-get install php-apc
+             
+PHP 5.5 replaces APC with Opcache. Opcache is bundled with PHP 5.5 so it should 
+not be necessary to install it separately. For reference, this command installs 
+it on CentOS/Red Hat/Fedora running PHP 5.5 and up:       
 
 .. code-block:: console
 
-	# yum install php-pecl-apc
+        $ sudo yum install php-pecl-zendopcache
+        
+For reference, this installs it on Ubuntu:
+
+.. code-block:: console
+
+        $ sudo apt-get install php5-xcache        
```

Comment:
XCache is not Opcache but i'm fine with the recommendation of php5-xcache. Had running this for a longer time here without any problems.